docs/security_bots.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Managing Bots
+tags:
+  - Security
+  - Bots
+---
+
+# Managing Bots
+
+A public-facing production instance will likely encounter bad bots and other malicious traffic that will consume resources. There are many solutions available that address a variety of different needs, but we provide basic configurations and a Docker image for integrating the [NGINX Ultimate Bad Bot & Referrer Blocker](https://github.com/mitchellkrogza/nginx-ultimate-bad-bot-blocker).
+
+## Deployment
+
+1. Uncomment or add the following docker-compose environment variables, replacing any appropriate values with your own (leave the blocker and cron disabled to start):
```

Comment:
Please be specific by pointing to the configs that keep blocked disabled, means  NGXBLOCKER_ENABLE=false  and NGXBLOCKER_CRON_START=false

Reply:
Resolved via 028942c.